.gitignore

```diff
@@ -0,0 +1 @@
+env
```

Comment:
When we generate the python `env`, thousands of files and generated. This should be local to our own dev environments, so we can ignore this entire directory in our version control.

Reply:
This is good I would just like to add two more things to the .gitignore: ".vscode", and "__pycache__" without quotations. The .vscode is to ignore certain configs that may be specific to your system and the pycache is just the generated build code.

Reply:
Sure, those files aren't part of this PR (or the repo, since it's all in the virtual env local to your computer) but this is good to add anyways going forward. Done.